src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -259,3 +259,75 @@ def validate_vnet_subnet_id(namespace):
         from msrestazure.tools import is_valid_resource_id
         if not is_valid_resource_id(namespace.vnet_subnet_id):
             raise CLIError("--vnet-subnet-id is not a valid Azure resource ID.")
+
+
+def validate_nodepool_labels(namespace):
+    """Validates that provided node labels is a valid format"""
+
+    if hasattr(namespace, 'nodepool_labels'):
+        labels = namespace.nodepool_labels
+    else:
+        labels = namespace.labels
+
+    if labels is None:
+        return
+
+    if isinstance(labels, list):
+        labels_dict = {}
+        for item in labels:
+            labels_dict.update(validate_label(item))
+        after_validation_labels = labels_dict
+    else:
+        after_validation_labels = validate_label(labels)
+
+    if hasattr(namespace, 'nodepool_labels'):
+        namespace.nodepool_labels = after_validation_labels
+    else:
+        namespace.labels = after_validation_labels
+
+
+def validate_label(label):
```

Comment:
Maybe you can add there rules to help message.

Reply:
The syntax for labels are too long. I included a link to the definition: https://aka.ms/node-labels in the help. That should help.

Reply:
Thank you!